README.md

```diff
@@ -1,75 +1,46 @@
-[<img align="right" src="https://img.shields.io/github/issues/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/issues)
-[<img align="right" src="https://img.shields.io/github/license/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/blob/master/LICENSE)
-[<img align="right" src="https://img.shields.io/github/forks/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/network/members)
-[<img align="right" src="https://img.shields.io/github/stars/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/stargazers)
-[<img align="right" src="https://img.shields.io/github/release/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/releases)
+[ ![Get Addon][amo-button] ][amo-website]
 
-![SUMO Live Helper](/src/res/logo/sumo64.png)
-# SUMO Live Helper [<img align="right" src="https://addons.cdn.mozilla.net/static/img/addons-buttons/AMO-button_2.png">](https://addons.mozilla.org/en-US/firefox/addon/sumo-live-helper-/)
-The goal of this Add-on is give to all volunteers of SUMO (Support Mozilla) a tool to see when there are new support questions in the forums that have not yet been answered. This will help to improve the Mozilla products user experience by getting answers to their problems more quickly.
+# ![SUMO Live Helper](src/res/logo/sumo32.png) SUMO Live Helper ![Version][version]
 
-![Interface of SUMO Live Helper](sumo-live-helper.png)
+A WebExtension to help SUMO contributors to quickly and easily find unanswered 
```

Comment:
Maybe "An extension to help SUMO contributors quickly and easily find unanswered questions on support.mozilla.org"

🤔

Reply:
Done.